test/integration/core/exceptions.tap.js

```diff
@@ -87,7 +87,7 @@ tap.test('Triggers harvest while in serverless mode', (t) => {
     payload += data.toString('utf8')
   })
 
-  proc.on('message', function(errors) {
+  proc.on('message', async function(errors) {
```

Comment:
i don't see anything async/promise related in here... so the `async` keyword seems like it will create an unnecessary promise?

Reply:
That was an artifact of an edit/revision, though I'd overlooked removing the async here. Thanks!